playground/Testcases.tsx

```diff
@@ -0,0 +1,114 @@
+import { useRef, useEffect, Fragment } from "react";
+
+import { FLOWCHART_DIAGRAM_TESTCASES } from "./testcases/flowchart";
+import { SEQUENCE_DIAGRAM_TESTCASES } from "./testcases/sequence.ts";
+import { CLASS_DIAGRAM_TESTCASES } from "./testcases/class.ts";
+import { UNSUPPORTED_DIAGRAM_TESTCASES } from "./testcases/unsupported.ts";
+
+import type { UpdateMermaidDefinition } from "./index.tsx";
+import { MermaidDiagram } from "./MermaidDiagram.tsx";
+
+interface TestCaseProps {
+  name: string;
+  testcases: { name: string; definition: string }[];
+  onChange: (definition: string, activeTestcaseIndex: number) => void;
+  error: string | null;
+  activeTestcase?: number;
```

Comment:
rename to `activeTestCaseIndex` as its the index and not the testcase

Reply:
done in [d3716a2](https://github.com/excalidraw/mermaid-to-excalidraw/pull/52/commits/d3716a234c903aaf4ce19ec5f610f3b090094ae7)